Brokerages/WebSocketWrapper.cs

```diff
@@ -82,18 +167,27 @@ public void Connect()
         /// </summary>
         public void Close()
         {
-            _wrapped.Close();
+            try
+            {
+                _wrapped.CloseAsync(WebSocketCloseStatus.NormalClosure, string.Empty, _cancellationTokenSource.Token);
```

Comment:
After closure could we continue to operate after a `Connect()`, is it a valid scenario? If so, `_cancellationTokenSource` should be reset?